dsp-shared/src/main/scala/dsp/valueobjects/LangString.scala

```diff
@@ -0,0 +1,79 @@
+package dsp.valueobjects
+
+import dsp.errors.ValidationException
+import zio.prelude.Validation
+
+/**
+ * LangString value object
+ */
+sealed abstract case class LangString private (language: LanguageCode, value: String)
+
+object LangString {
+
+  /**
+   * Creates a [[zio.prelude.Validation]] that either fails with a ValidationException or succeeds with a LangString value object.
+   *
+   * @param language a [[LanguageCode]] value object representing the language of the LangString.
+   * @param value    the string value of the LangString.
+   * @return a Validation of a LangString value object.
+   */
+  def make(language: LanguageCode, value: String): Validation[ValidationException, LangString] =
+    if (value.isBlank()) {
+      Validation.fail(ValidationException(LangStringErrorMessages.LangStringValueEmpty))
+    } else {
+      Validation.succeed(new LangString(language, value) {})
+    }
+
+  /**
+   * Creates a [[zio.prelude.Validation]] that either fails with a ValidationException or succeeds with a LangString value object.
+   *
+   * @param language a two-digit language code string representing the language of the LangString.
+   * @param value    the string value of the LangString.
+   * @return a Validation of a LangString value object.
+   */
+  def makeFromStrings(language: String, value: String): Validation[ValidationException, LangString] =
+    for {
+      languageCode <- LanguageCode.make(language)
+      langString   <- LangString.make(languageCode, value)
+    } yield langString
+
+  /**
+   * Unsafely creates a LangString value object.
+   * Warning: skipps all validation. Should not be used unless there is no possibility for the data to be invalid.
+   *
+   * @param languagea [[LanguageCode]] value object representing the language of the LangString.
+   * @param value     the string value of the LangString.
+   * @return a LanguageCode value object
+   */
+  def unsafeMake(language: LanguageCode, value: String): LangString =
```

Comment:
Do you actually use this somewhere? If not, I would not implement it.

Reply:
you're right, it's not used anymore. I'll delete it

Reply:
actually, no, I use it after all... I put it on the "to discuss" list.